src/prefect/client/base.py

```diff
@@ -361,9 +361,16 @@ async def _add_csrf_headers(self, request: Request):
             try:
                 token_response = await self.send(token_request)
             except PrefectHTTPStatusError as exc:
-                if exc.response.status_code == status.HTTP_404_NOT_FOUND:
-                    # The token endpoint is not available, likely an older
-                    # server version; disable CSRF support
+                old_sever = exc.response.status_code == status.HTTP_404_NOT_FOUND
```

Comment:
```suggestion
                old_server = exc.response.status_code == status.HTTP_404_NOT_FOUND
```

Reply:
you might have sever ties with the variable name elsewhere too

Reply:
Thank you!